lib/api/CDetectionRulesJsonParser.cc

```diff
@@ -153,6 +170,8 @@ bool CDetectionRulesJsonParser::parseRuleActions(const json::object& ruleObject,
             action |= model::CDetectionRule::E_SkipResult;
         } else if (parsedAction == SKIP_MODEL_UPDATE) {
             action |= model::CDetectionRule::E_SkipModelUpdate;
+        } else if (parsedAction == FORCE_TIME_SHIFT) {
```

Comment:
Handling for callback action?

Reply:
I removed `model::CDEtectionRule::E_Callback` instead. I will use specific callback action enums instead of a generic one.